packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -0,0 +1,143 @@
+   */
+  @Prop({mutable: true, reflect: true}) public facetId?: string;
+  /**
+   * The field whose values you want to display in the facet.
+   */
+  @Prop({reflect: true}) public field!: string;
+  /**
+   * The non-localized label for the facet.
+   */
+  @Prop({reflect: true}) public label = 'no-label';
+  /**
+   * Whether to exclude the parents of folded results when estimating the result count for each facet value.
+   */
+  @Prop({reflect: true}) public filterFacetCount = true;
+
+  @Prop() public injectionDepth = 1000;
+  /**
+   * The number of values to request for this facet.
+   * Also determines the number of additional values to request each time more values are shown.
+   */
+  @Prop({reflect: true}) public numberOfValues = 6;
+  /**
+   * The sort criterion to apply to the returned facet values.
+   * Possible values are 'score', 'alphanumeric', 'occurrences', and 'automatic'.
+   */
+  @Prop({reflect: true}) public sortCriteria: FacetSortCriterion = 'automatic';
+
+  public initialize() {
+    this.searchStatus = buildSearchStatus(this.bindings.engine);
+    const options: FacetOptions = {
+      facetId: this.facetId,
+      field: this.field,
+      numberOfValues: this.numberOfValues,
+      sortCriteria: this.sortCriteria,
+      facetSearch: {numberOfValues: this.numberOfValues},
+      filterFacetCount: this.filterFacetCount,
+      injectionDepth: this.injectionDepth,
+    };
+    this.facet = buildFacet(this.bindings.engine, {options});
+    this.facetId = this.facet.state.facetId;
+    registerFacetToStore(this.bindings.store, 'facets', {
+      label: this.label,
+      facetId: this.facetId!,
+      element: this.host,
+    });
+  }
+
+  private renderValuesContainer(children: VNode[]) {
+    const classes = `mt-3 ${'box-container'}`;
+    return (
+      <ul part="values" class={classes}>
+        {children}
+      </ul>
+    );
+  }
+
+  private renderValue(facetValue: FacetValue, onClick: () => void) {
+    const displayValue = getFieldValueCaption(
+      this.field,
+      facetValue.value,
+      this.bindings.i18n
+    );
+    const isSelected = facetValue.state === 'selected';
+
+    return (
+      <FacetValueBox
+        displayValue={displayValue}
+        numberOfResults={facetValue.numberOfResults}
+        isSelected={isSelected}
+        i18n={this.bindings.i18n}
+        onClick={onClick}
+        searchQuery={this.facetState.facetSearch.query}
+      >
+        <FacetValueLabelHighlight
+          displayValue={displayValue}
+          isSelected={isSelected}
+          searchQuery={this.facetState.facetSearch.query}
+        ></FacetValueLabelHighlight>
+      </FacetValueBox>
+    );
+  }
+
+  private renderValues() {
+    return this.renderValuesContainer(
+      this.facetState.values.map((value) =>
+        this.renderValue(value, () => this.facet.toggleSelect(value))
+      )
+    );
+  }
+
+  public render() {
+    return <FacetContainer>{this.renderValues()}</FacetContainer>;
```

Comment:
This puts a border around the component while the segmented facet's title can be outside, also the rounded border is a little less rounded...
I'd remove the FacetContainer for now and see later when the styling is added
<img width="652" alt="Screen Shot 2022-05-19 at 3 44 22 PM" src="https://user-images.githubusercontent.com/4923043/169389704-e2b92c42-8f7e-4af5-9f80-110a8980d204.png">